src/Symfony/Component/Serializer/Normalizer/SerializerAwareNormalizer.php

```diff
@@ -11,26 +11,17 @@
 
 namespace Symfony\Component\Serializer\Normalizer;
 
-use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Serializer\SerializerAwareTrait;
 use Symfony\Component\Serializer\SerializerAwareInterface;
 
 /**
  * SerializerAware Normalizer implementation.
  *
  * @author Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * @deprecated since version 3.1, to be removed in 4.0. Use the SerializerAwareTrait instead.
```

Comment:
> You must trigger a deprecation error

Since there is no more code / method in this class i don't think it's possible, i have done this deprecated feature in the same way it was done for the ContainerAwareTrait in 2.8 https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/DependencyInjection/ContainerAware.php

> You should also adapt existing normalizers in Symfony to avoid using this deprecated class

It's already done, but maybe i have miss some implementations ?